.github/actions/install-wdk/action.yml

```diff
@@ -26,6 +26,7 @@ runs:
         $targetArch = '${{ inputs.target }}'
         switch ($targetArch) {
             'amd64' { $targetArch = 'x64' }
+            'x64' { $targetArch = 'x64' }
```

Comment:
Why the redundancy here? should be consistent across the repo

Reply:
I made architecture names consistent across all the workflows and removed this redundant case as well.